src/main/java/io/quarkus/gizmo2/Constant.java

```diff
@@ -95,6 +95,54 @@ static Constant of(Enum.EnumDesc<?> value) {
         return ConstantImpl.of(value);
     }
 
+    /**
+     * {@return a constant for the given value}
+     * @param value the value to create a constant from (must not be {@code null})
+     */
+    static Constant of(Byte value) {
+        return ConstantImpl.of(value);
+    }
+
+    /**
+     * {@return a constant for the given value}
+     * @param value the value to create a constant from
+     */
+    static Constant of(byte value) {
+        return ConstantImpl.of((Byte) value);
```

Comment:
These methods could just be `ConstantImpl.of(value)` right?

Reply:
Good point, will fix.

Reply:
Done.